ts_js_cpp_c/cPart.h

```diff
@@ -0,0 +1,14 @@
+#ifndef FOO_H
+#define FOO_H
+
```

Comment:
Myabe here you could use:
```c
#ifndef _C_PART_H_
#define _C_PART_H_
```

Reply:
I applied this suggestion, thank you :+1: